libs/assertion/include/hpx/assertion.hpp

```diff
@@ -58,16 +58,27 @@ namespace hpx { namespace assertion {
             __FILE__, __LINE__, HPX_ASSERT_CURRENT_FUNCTION},                  \
         HPX_PP_STRINGIZE(expr), [&]() { return msg; });                        \
 /**/
+#define HPX_ASSERT_EQ(x, count)
+#define HPX_ASSERT_EQ_(x, count)
+    ::hpx:assertion::detail::evaluate_count_assert(
```

Comment:
`evaluate_count_assert` isn't defined anywhere. Did you maybe forget to add it?

Reply:
Should be `evaluate_assert`.